### PR TITLE
✍️ Add the bounty schema

### PR DESCRIPTION
**Pull request by thesan**
~~Require #1990~~

It adds the first draft for the schema from https://github.com/Joystream/joystream/pull/2945 and part of the some mocks.

~~Note that `BountyEntry` is currently wrong (it should be fixed in https://github.com/Joystream/joystream/pull/2945). Also the events and the forum thread is missing from the mocks.~~

### CODE REVIEW

**Comment by thesan** [comment]
@Qcza @RafalPloszka would it be okay to base some of the bounty development on this branch while waiting for #1990 and this PR to be merge ?

**Comment by RafalPloszka** [comment]
> @Qcza @RafalPloszka would it be okay to base some of the bounty development on this branch while waiting for #1990 and this PR to be merge ?

@thesan I think it should be fine :)

File: packages/ui/dev/query-node-mocks/generators/generateBounties.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  const entries = repeat(generateEntry(mocks, bounty), randomFromRange(0, 5))
+
+  return {
+    bounties: [...data.bounties, bounty],
+    bountyContributions: [...data.bountyContributions, ...contributions],
+    bountyEntries: [...data.bountyEntries, ...entries],
+  }
+}
+
+const generateContribution = (mocks: DependOnMocks, bounty: RawBountyMock) => (
```
Comment (Qcza) [inline packages/ui/dev/query-node-mocks/generators/generateBounties.ts:101]: unused argument
```suggestion
const generateContribution = (bounty: RawBountyMock) => (
```

Comment (thesan) [reply packages/ui/dev/query-node-mocks/generators/generateBounties.ts:101]: Thanks I missed that. But mostly it means that I forgot to add a `contributor: Membership!` field in the schema :facepalm:

**Review by RafalPloszka — APPROVED** [review]

**Pull request merged**